roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
ah ok, these are bug fixes. But is the logic broken? We will still need `contiv-compose --version` to return correct version which we don't seem to be doing today.